connect_ext_test.go

```diff
@@ -503,6 +504,42 @@ func TestBidiRequiresHTTP2(t *testing.T) {
 	)
 }
 
+func TestCompressMinBytes_Enabled(t *testing.T) {
```

Comment:
Isn't this testing both compressed and uncompressed, the `_Enabled` suffix seems redundant? There might be a good reason though.

Reply:
By default CompressMinBytes is set to 0 which effectively disables it. I can remove that though.